### DISCUSSION TITLE
一些问题

### DISCUSSION THREAD

**Discussion by mihuru** [HMBSbige/BilibiliLiveRecordDownLoader#238] [General]
1. 下载别的文件能达到20多mb/s，然而直播的下载速率只有1mb/s，下载得到的文件断断续续的
2. 直播间正在直播但显示未开播
3. 直播结束却显示录制中

**Comment by HMBSbige** [comment]
emmmm，从你的描述可以看出你不是很懂网络，我只能说大概率是你自己的网络问题
